### DISCUSSION TITLE
Suddenly getting null values on shop data

### DISCUSSION THREAD

**Discussion by emergingdzns** [etsy/open-api#1279] [Q&A]
All of a sudden today, we are getting null values returned for almost every field in the `/api/shops/{shop_id}` endpoint. This is throwing off our script of course. The fields in question are:
```
shop_location_country_iso
transaction_sold_count
create_date
active_listings
shipping_from
digital_listing_count
review_count
review_average
user_id
created_timestamp
currency_code
is_vacation
update_date
listing_active_count
```

There may be more but this is what I'm finding in our error logs readily. And they are flooding in.

I can't tell for certain if it's ALL shops or just some. But it's quite prevalent.

Anyone else having these issues?

Can you check on this Etsy?
@mjec @nateswart ?

**Comment by kbasali** [comment]
![etsy-shop](https://github.com/user-attachments/assets/3e36719e-a1ad-49a2-b31a-c309b031a71f)
@emergingdzns  It works.
